src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsbookavideolinkapi/common/StringExtensionUtilsTest.kt

```diff
@@ -39,4 +40,15 @@ class StringExtensionUtilsTest {
       assertThat(address.isEmail()).isEqualTo(true)
     }
   }
+
+  @Test
+  fun `should recognise various UK phone numbers`() {
+    listOf(
+      "+44 123 456 7890",
+      "07928 660553",
+      "0344 561 6789",
+    ).forEach { phoneNumber ->
+      phoneNumber.isUkPhoneNumber() isBool true
+    }
```

Comment:
Maybe some bad ones here too? 999,  0247-HELLO-YOU?

Reply:
Turns out the library by allows alpha numbers e.g. 0800 800 ACME, the characters would be converted to numbers.  I have added a further check to prevent this.